src/ServiceFabric/ServiceFabric/Commands/ServiceFabricClusterCertificateCmdlet.cs

```diff
@@ -97,6 +97,11 @@ public abstract class ServiceFabricClusterCertificateCmdlet : ServiceFabricClust
         [ValidateNotNullOrEmpty]
         public string SecretIdentifier { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipeline = true, ParameterSetName = ByExistingKeyVault,
+                   HelpMessage = "The thumbprint for the certificate corresponding to the SecretIdentifier. Use this if the certificate is not managed as the key vault would only have the certificate stored as a secret and the cmdlet is unable to retrieve the thumbprint.")]
```

Comment:
>Use this if the certificate is not managed as the key vault would only have the certificate stored as a secret and the cmdlet is unable to retrieve the thumbprint [](start = 108, length = 162)

Use this when:
- you do not have permissions to retrieve the certificate from the vault
- the certificate is not managed by KeyVault (was imported as a secret)
- you have the thumbprint or simply do not want to retrieve the certificate from the vault

Reply:
ok thanks, I'll change the message in a new pr

---
In reply to: [418230302](https://github.com/Azure/azure-powershell/pull/11758#discussion_r418230302) [](ancestors = 418230302)